dspace-xmlui/src/main/webapp/themes/UFAL/lib/xsl/aspect/artifactbrowser/item-view.xsl

```diff
@@ -943,7 +943,7 @@
 									<xsl:copy-of select="mets:Local/mets:file"/>
 								</xsl:variable>  						  							
   								<xsl:choose>
-  									<xsl:when test="@MIMETYPE='application/zip'">
+  									<xsl:when test="@MIMETYPE='application/zip' or @MIMETYPE='application/x-tar'">
 			  							<xsl:value-of select="ft:parse($files)" disable-output-escaping="yes" />
```

Comment:
It behaves differently on archives and plain text, maybe we could make the archive behavior the default, or move the logic elsewhere. I would leave it right now and rethink this when we add new previews (or the rest of the archives)